app/views/users/show.html.erb

```diff
@@ -8,7 +8,23 @@
     <% end %>
     <div class="d-flex flex-column justify-content-center mr-3 ml-3">
       <h2><%= "#{@user.first_name} #{@user.last_name}" %></h2>
-      <p>4th year Software Engineering Stduent</p>
+      <p>
+        <% if @user.role == "student"%>
+          <% if @user.year == "1"%>
+            <%= "#{@user.year}st year student in #{@user.faculty}"%>
+          <%elsif @user.year == "2"%>
+            <%= "#{@user.year}rd year student in #{@user.faculty}"%>
+          <%elsif @user.year == "3"%>
+            <%= "#{@user.year}rd year student in #{@user.faculty}"%>
+          <%elsif @user.year == "4"%>
+            <%= "#{@user.year}st year student in #{@user.faculty}"%>
```

Comment:
Ruby has another way to convert numbers to 
https://apidock.com/rails/ActiveSupport/Inflector/ordinalize